tests/unit/SigningParty.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { createPinia, setActivePinia } from 'pinia'
+import { mount } from '@vue/test-utils'
+import SigningParty from '@/components/SpecialResolution/SigningParty.vue'
+
+setActivePinia(createPinia())
```

Comment:
Why do you need this ^^ line but not the following line?
`const store = useStore()`

Reply:
Because I'm not using the store in the unit test, but need Pinia to mount the component.

Reply:
Could you please add a comment above this line saying that? Thanks.

Reply:
Thanks for the comment. It will prevent a future dev (maybe me) from deleting what looks like unused code 😁